README.md

```diff
@@ -53,23 +58,38 @@ The [base schema](metadata-schema/baseSchema.schema.json) defines the supported
           "tdrFileHeader": "ClientSideFileLastModifiedDate"
         }
       ],
-      "message": {
-        "format": "Date last modified must be a valid date"
-      },
-    }
-.....
+    },
+    "end_date": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "format": "date",
+      "alternateKeys": [
+        {
+          "tdrFileHeader": "Date of the record"
+        }
+      ],
+      "daBeforeToday": "Validates that end date is earlier than today's date"
```

Comment:
Is `"Validates that end date is earlier than today's date"` present here just to make the property truthy?

Reply:
yes